package.json

```
@@ -21,31 +21,53 @@
   },
   "dependencies": {
     "@hookform/resolvers": "^5.1.1",
+    "@internationalized/date": "^3.8.2",
```

Biome (2.1.2)

[error] The key @internationalized/date was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)